docs/dpdk/configMap-all-nics.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sriovdp-config
+  namespace: kube-system
+data:
+  config.json: |
```

Comment:
ah, or you could simply get the readme to point to here if you want to use selectors as a slice instead?